### PR TITLE
adding colab ipynb

### PR DESCRIPTION
**Pull request by johnsolk**
Adding tutorial script ipynb for CoLab link

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,3 +90,7 @@ library("loupeR")
 # print extra debug information
 create_bugreport_from_seurat(seurat_obj)
 ```
+
+## Tutorials
```
Comment (esiegel) [inline README.md:94]: Can you move this section above the `Troubleshooting` section?  Also, can you add a link to this section in the top part at line 12?

File: README.md [hunk 2]
```diff
@@ -90,3 +90,7 @@ library("loupeR")
...
+
+An [interactive script]() with an example dataset can be run from this notebook here:
```
Comment (esiegel) [inline README.md:96]: You need to update the link to `misc/tutorials/5k_mouse_brain.ipynb`.  Also, I would update this to be a markdown list.  Something like:

```markdown
- [mouse brain notebook](misc/tutorials/5k_mouse_brain.ipynb): basic processing of a 10x dataset with export to a Loupe file using `loupeR`
```

**Review by esiegel — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (johnsolk) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (johnsolk) [reply README.md]: Done.

**Comment by johnsolk** [comment]
hi @esiegel thanks for comments! the README is edited and some edits on the .ipynb as well. Can you please take a look and merge if ready? Let me know if more changes needed. 

Once merged and .ipynb in the public-facing repo, I can generate a public Google CoLab link and then will need to edit the README again adding a CoLab button/link

**Review by esiegel — APPROVED** [review]
LGTM

**Pull request merged**